src/components/FilterBar/index.tsx

```diff
@@ -1,23 +1,41 @@
-import React from 'react';
-import DropdownFilter from '../DropdownFilter';
-import SearchFilter from '../SearchFilter';
-import { DropdownFilterConstant } from '../../constants'; 
-import './FilterBar.css';
+import React, { useEffect, useState } from "react";
+import DropdownFilter from "../DropdownFilter";
+import SearchFilter from "../SearchFilter";
+import { getFilters as getFiltersHelper } from "../../helpers/GetFilters";
+import "./FilterBar.css";
+import { DropDownData } from "../../interfaces";
 
 const FilterBar = () => {
-    return (
+  const [dropDownData, setDropDownData] = useState<DropDownData>(
+    {} as DropDownData
+  );
```

Comment:
`{} as DropDownData` why is this required?

Reply:
{} will throw type error as it is not type of DropDownData

Reply:
your state type should be `DropDownData | null`

Reply:
your state type should be DropDownData | null